src/plugins/intel_gpu/src/graph/mvn.cpp

```diff
@@ -20,7 +20,27 @@ layout mvn_inst::calc_output_layout(mvn_node const& node, kernel_impl_params con
         output_type = data_types::f32;
     }
 
-    return layout(output_type, input_node_layout.format, input_node_layout.get_tensor());
+    auto output_format = input_node_layout.format;
+    auto input_psahpe = input_node_layout.get_partial_shape();
+    auto prim = node.get_primitive();
+
+    // When it needs flattening, use the plain format
+    // if the reduction axis is not a blocked axis.
+    if (output_format.is_blocked() && prim->requires_alignment(input_psahpe)) {
+        int64_t reduce_axis = format::block_sizes(output_format)[0].first;
```

Comment:
Seems that reduction_axis is confused as output format's block axis. 
reduction axis is defined in primitive.

Reply:
Also I think format change should be done in layout_optimizer::get_preferred_format